### ISSUE TITLE
Matplotlib produces blank eps images when text.usetex option is True in matplotlibrc

### ISSUE THREAD

**Issue by latrop** [matplotlib/matplotlib#17373]
<!--To help us understand and resolve your issue, please fill out the form to the best of your ability.-->
<!--You can feel free to delete the sections that do not apply.-->

### Bug report

When I try to save a plot as an EPS file with savefig() function, the result is just blank image. If I set text.usetex option to False, everything looks fine. The problem is only with EPS format. PDF or PNG files are produced correctly regardless of text.usetex option.

<!--A short 1-2 sentences that succinctly describes the bug-->

**Code for reproduction**
<!--A minimum code snippet required to reproduce the bug.
Please make sure to minimize the number of dependencies required, and provide
any necessary plotted data.
Avoid using threads, as Matplotlib is (explicitly) not thread-safe.-->
The code can be as simple as
```python
from matplotlib import pyplot as plt                                                                                                   
                                                                                                                                       
plt.plot([1, 2], [1, 2])                                                                                                               
plt.savefig("test.eps") 
```

**Actual outcome**
Blank EPS file (but it's size is 182K). 

**Expected outcome**
EPS file containing an actual plot.

**Matplotlib version**

  * Operating system: Arch linux, everything is up to date
  * Matplotlib version: 3.2.1, installed using Arch package manager
  * Matplotlib backend: Qt5Agg
  * Python version: 3.8.2

**Comment by anntzer** [comment]
I cannot repro with the Arch package.

**Comment by cnvega** [comment]
I'm having exactly the same issue since a couple of weeks (Arch Linux as well). When the text.usetex option is set to True, the eps figures result in blank files. I tried the agg and ps backends. My packages are:
python 3.8.2-2
python-matplotlib 3.2.1-1

**Comment by anntzer** [comment]
Ah yes, sorry I missed it.  This is however fixed as of master by https://github.com/matplotlib/matplotlib/pull/17255, AFAICT, which is already backported to the (not yet released) 3.2.2.
